### PR TITLE
fix: show suggestions also when value is null

### PR DESCRIPTION
**Pull request by Wurper**
This fixes the issue that the suggestions does not show until a input has been made and value is not null.
no ref.

### CODE REVIEW

**Review by dotpointer — APPROVED** [review]
Tested in Substitutes, worked as usual there.

File: paper-autocomplete-suggestions.js [hunk 1]
```diff
@@ -534,7 +534,7 @@ Polymer({
 		this._currentIndex = -1;
 		this._scrollIndex = 0;
 
-		let value = event.target.value;
+		let value = event.target.value ? event.target.value : '';
```
Comment (cristinecula) [inline paper-autocomplete-suggestions.js]:
```suggestion
		let value = event.target.value || '';
```

**Review by cristinecula — CHANGES_REQUESTED** [review]
LGTM, but I don't know where to test this in frontend

**Comment by dotpointer** [comment]
> LGTM, but I don't know where to test this in frontend

Me neither, I tested on non-null locations.

**Comment by Wurper** [comment]
Well, I did another PR in frontend for replacing drop-down-menus for paper-autocomplete in the sales/customer/view. https://github.com/Neovici/cosmoz-frontend/pull/1879
And those had these issues.

**Pull request merged**